src/index.css

```diff
@@ -38,27 +38,28 @@ code {
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background: #fafafa;
+  background-color: #fafafa;
   margin-top: 8.3rem;
+  width: 87.5rem;
 }
 
 /* navbar */
 nav {
   display: flex;
```

Comment:
- [ ] You have used `display: flex` 9 times, which is bad css practice because you are repeating yourself a lot, I would suggest you create a class with display:flex in it and give it to all html elements that need it 
![image](https://github.com/0sugo/bookstore/assets/62705947/c58b8b9e-e1da-48b2-8c31-9a114743314b)